### ISSUE TITLE
helm-addon.kubectl-provider.ts : string replacement data lost

### ISSUE THREAD

**Issue by laptua** [aws-quickstart/cdk-eks-blueprints#460]
### Describe the bug

The code:
https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/lib/addons/helm-addon/kubectl-provider.ts

```   
    public static applyManifestTemplate = function(document: any, values: Values) : any {
        const valueMap = new Map(Object.entries(values));
        let data = JSON.stringify(document);
        valueMap.forEach((value: string, key: string) => {
            data.replace(`{{${key}}}`, value);
        });
        return JSON.parse(data);
    };
```

should be:
```
          data=data.replace(`{{${key}}}`, value);
```

### Expected Behavior

string replacement

### Current Behavior

string replacement do not occur

### Reproduction Steps

see code above

### CDK CLI Version

2.25.0 (build ae1cb4b)

### EKS Blueprints Version

1.1.0

### Node.js Version

v14.19.3

### Environment details (OS name and version, etc.)

mac

**Comment by shapirov103** [comment]
Thank you for catching it. This was fixed in #445 
[link](https://github.com/aws-quickstart/cdk-eks-blueprints/pull/445/files#diff-c75c106238e568ef4d96e8b8743315140e44574d56e1aeabcbeba836d886d49aR126)

**Comment by shapirov103** [comment]
@laptua can you verify when you get a chance on 1.2.0? We can reopen if you observe any issues.
